api/proto/metrics/metrics.proto

```diff
@@ -521,3 +523,8 @@ message logfileMetrics {
 message zedboxStats {
   uint32 numGoRoutines = 1;
 }
+
+message vlanInfo {
```

Comment:
I would add some comments explaining what num_trunk_ports is and  more importantly what the key and value are for the map.

Reply:
done